components/molecules/BenefitCardHeaderSummary.js

```diff
@@ -0,0 +1,33 @@
+import propTypes from 'prop-types'
+import { SummaryTypes } from '../../objects/UniversalBenefit'
+
+export default function BenefitCardHeaderSummary(props) {
+  const t = props.locale
+  let typesWithLinks = [SummaryTypes.PaymentAmount, SummaryTypes.LatestStatus]
+
+  return (
+    <div>
+      <p>{t[props.summary.type].title}</p>
+      <p className="font-bold">
+        {props.summary.value}
+        <br />
```

Comment:
`<br />` should be avoided for a11y issues.

Reply:
yes and no.... but pretty sure you're right in this case